ts/useIsPortrait.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react"
+import { Dimensions } from "react-native"
```

Comment:
I saw in the docs that [useWindowDimensions](https://reactnative.dev/docs/usewindowdimensions) is preferred, and it seems like a good way for the code to rotate dynamically. Should we use that here?

Reply:
I will use useWindowDimensions in useIsPortrait hook. I will rename the useIsPortrait hook to useScreenDimensions hook. I will calculate and export the height and width within the useScreenDimensions hook.